scripts/document_types/trait.py

```diff
@@ -285,6 +293,7 @@ def __get_descendants(efo_data):
 
 
     for row in tqdm(efo_data, desc='Getting EFO term - hierarchicalDescendants mapping'):
+        print >> sys.stderr, '\nEFO term: ', row[2], row[4]
```

Comment:
This is not particularly future proof. `sys.stderr.write("message")` works on Python3 as well.

Reply:
That was there to debug, not intended for production code.